llama-index-packs/llama-index-packs-code-hierarchy/examples/CodeHierarchyNodeParserUsage.ipynb

```diff
@@ -0,0 +1,1180 @@
+{
```

Comment:
<p>It would be ideal to use a llama-index agent IMO</p><p> </p><p>I could take a stab at doing that</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/run-llama/llama_index/pull/10671/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='41'/>

Reply:
<p>True, I didn't know llama_index had agents of its own.</p>

Reply:
<p>I did notice you have to run the agent several times to get a high quality response like this one. Sometimes it stops reading too early.</p>

Reply:
I'll take a stab at implementing this in the notebook with fully llama-index and see how it goes 👀